### PR TITLE
Icrc receive modal

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Implement CkETH receive modal feature.

# Changes

- refactor and pass additional parameters to `ReceiveButton` and related modal to make the receive modal generic for icrc
- implement the parameters (either constant for ICRC or those of the universe)
- rename `SnsReceiveModal` to `IcrcReceiveModal`
- add modal for Icrc univeser on Accounts page
- adapt arial label for logo for icrc

### CODE REVIEW

File: frontend/src/lib/types/icrc-accounts.modal.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import type { UniverseCanisterId } from "$lib/types/universe";
 import type { IcrcTokenMetadata } from "./icrc";
 
+// TODO: to be refactored:
```
Comment (lmuntaner) [inline frontend/src/lib/types/icrc-accounts.modal.ts:4]: Add a reference to GIX-2150 to easily find the places that need cleaning.

File: frontend/src/lib/routes/Accounts.svelte [hunk 2]
```diff
@@ -143,9 +143,10 @@
 
   {#if $isCkBTCUniverseStore}
     <CkBTCAccountsModals />
-  {:else if $selectedIcrcTokenUniverseIdStore}
-    <IcrcTokenAccountsModals />
   {:else}
+    {#if $selectedIcrcTokenUniverseIdStore}
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply frontend/src/lib/routes/Accounts.svelte:147]: Sure, added the number in the `icrc-accounts.modal` todo

File: frontend/src/lib/types/icrc-accounts.modal.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/types/icrc-accounts.modal.ts]: Sure, done

**Review by lmuntaner — APPROVED** [review]
LGTM!

Just minor comments to find out cleaning opportunities later.

**Pull request merged**